api/src/controllers/userController.js

```diff
@@ -0,0 +1,34 @@
+const jwt = require('jsonwebtoken');
+
+const { User } = require('../models/models');
+
+const loginUser = async (req, res) => {
+  const { email, password } = req.body;
+  const user = await User.findOne({ where: { email } });
+  if (!user) {
+      return res.status(401).json({
+        message: 'Authentification failed. User not found',
+      });
+    }
+  if (user.password === password) {
+    const payload = { id: user.id };
+    const token = jwt.sign(payload, process.env.SECRET_KEY);
+    return res.json({
+      message: 'Success!', user, token,
+    });
+  }
```

Comment:
We write all business logic in services, but not in controller

Reply:
done)